scss/forms/_form-control.scss

```diff
@@ -192,6 +192,26 @@ textarea {
   width: $form-color-width;
   height: $input-height;
   padding: $input-padding-y;
+  border-color: var(--#{$prefix}body-color);
+
+  // Boosted mod
+  &:not(:disabled):hover {
```

Comment:
Not sure that the `:not(:disabled)` is mandatory here.

Reply:
Not mandatory anymore since we found a way to change the background color for disabled state.